src/adapters/search_suggestions_adapter.js

```diff
@@ -23,10 +23,10 @@ const SearchSuggestionsAdapter = State.extend({
         this.engine = data.engine;
         this.searchTerm = data.results.term;
 
-        this.remoteSuggestions.reset(_c.collect(data.results.remote, function (t) {
+        this.remoteSuggestions.reset(_c.collect(data.results.remote, function(t) {
           return { term: t, type: 'remote' };
         }));
-        this.localSuggestions.reset(_c.collect(data.results.local, function (t) {
+        this.localSuggestions.reset(_c.collect(data.results.local, function(t) {
```

Comment:
These anon-functions throw warnings if we tweak the `func-names` ESLint rule:

```
/Users/pdehaan/dev/github/universal-search-content/src/adapters/search_suggestions_adapter.js
  26:70  warning  Missing function expression name  func-names
  29:68  warning  Missing function expression name  func-names
```

Me thinks we can either give these a function name, or convert them to fat arrows.